documents/motions/10.2022.3 - Disciplinary Committee.md

```diff
@@ -1,34 +1,40 @@
+         2. The investigations can be initiated by the WCA Integrity Committee itself or by the WCA Board.
+      2. Performing independent investigations regarding WCA Regional Organizations.
+         1. The subjects of the investigations are incidents during WCA Competitions, incidents related to WCA Competitions, incidents related to the function of the Regional Organizations or incidents within the WCA Community. Incidents are alleged violations of the Spirit, Mission, Motions, Policies, Codes and/or Regulations of the WCA.
+         2. The investigations can be initiated by the WCA Integrity Committee itself or by the WCA Board.
+      3. Delivering an independent report on the investigations.
       4. Making decisions on disciplinary actions in relation to the investigated incidents. See [WCA Motion : Suspensions and other Sanctions](wcadoc{documents/motions/15.2022.1 - Suspensions and other Sanctions.pdf}).
    2. Confidant
-      1. Being the confidential contact for Registered Speedcubers and WCA Staff, in case of important personal matters that are related to WCA activities.
+      1. Being the confidential contact for Registered Speedcubers and WCA Staff, in case of important disciplinary and ethical matters that are related to WCA activities.
    3. Information and administration
       1. Informing the WCA Board on investigations, decisions, and disciplinary actions.
-      2. Publishing public information on the investigations, decisions, and disciplinary actions.
+      2. Publishing public information on the investigations, decisions, and disciplinary actions where necessary.
       3. Managing and administrating investigations, reports, decisions, and disciplinary actions.
       4. Keeping all information confidential that needs to stay confidential, based on internal and external guidelines
-2. All WCA Staff must fully cooperate with investigations by the WCA Disciplinary Committee.
-3. All Registered Speedcubers must fully cooperate with investigations by the WCA Disciplinary Committee.
-   1. All Registered Speedcubers should respond to emails sent on behalf of the WCA Disciplinary Committee within one week of each message.
-      1. A Registered Speedcuber must inform the WCA Disciplinary Committee or the WCA Board if they decide to stop responding to the WCA Disciplinary Committee and explain as to the reasons why.
-4. The WCA Disciplinary Committee will take the role of WCA Ethics Committee in case the WCA Ethics Committee is not constituted or for any reason non-functional.
-5. In case the WCA Disciplinary Committee is not constituted or for any reason non-functional, the WCA Ethics Committee will act as WCA Disciplinary Committee.
+   4. Document maintenance
+      1. Maintaining and publishing a Code of Ethics that all WCA Staff must abide by.
+      2. Maintaining and publishing a Code of Conduct that all WCA Members must abide by.
```

Comment:
all WCA Members? Or do we want it to apply to the WCA Community?

Reply:
It may be difficult to enforce the Code of Conduct on spectators of competitions however you are right it probably needs to apply to users on WCA online services

Reply:
The enforcement on spectators is something that we've discussed with our legal counsel. We are still awaiting their proposal, but they agreed that there's nothing necessarily that the WCA itself can enforce. Instead, this responsibility is better fit within the scope of the local RO/organizers and/or venue.